src/frontend/src/utils/userIntent.ts

```diff
@@ -15,13 +15,25 @@ export const intentFromUrl = (url: URL): UserIntent => {
   }
 };
 
-export const verbFromIntent = (intent: UserIntent): string => {
+export const authenticateIntent = (intent: UserIntent): string => {
   switch (intent.kind) {
     case "addDevice":
-      return "add your new device to your Internet Identity";
+      return "Authenticate to add your new device";
     case "auth":
-      return "authenticate using your Internet Identity";
+      return "Authenticate using Internet Identity";
     case "manage":
-      return "manage your Internet Identity";
+      return "Authenticate using Internet Identity";
+  }
+};
+
+// TODO: Remove me
```

Comment:
Should this code be removed or should the TODO be associated to a Jira ticket?

Reply:
We should probably make a JIRA ticket to deal with the follow-up changes and cleanup we need to do after the terminology changes. We should mention this TODO in that ticket.

Reply:
I created [II-36](https://dfinity.atlassian.net/browse/II-36) for this. Ideally, IMO all TODOs that we commit in the future should have the form
```
// TODO (II-??): Blah
```
and refer to a Jira ticket. This also conforms with the [guidelines](https://docs.google.com/document/d/1AQQGnvUU05kPg8_MahJtbyJ2KlPIPYbFblHNR4B3I9A/edit) we had to follow for shipshaping code for launch.